### PR TITLE
Grub Glide game

### PR DESCRIPTION
**Pull request by Exetric**
I made my first Gameboy game and I would like to see it uploaded it HomebrewHub cause it's basically where all Gameboy homebrew is hosted, thanks

### CODE REVIEW

File: entries/grub-glide/game.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            "playable": true
+        }
+    ],
+    "license": "Zlib",
+    "platform": "GB",
+    "repository": "https://github.com/Exetric/GrubGlide",
+    "screenshots": [
+        "1.png",
+        "2.png",
+        "3.png",
```
Comment (avivace) [inline entries/grub-glide/game.json:16]: I think this trailing comma should be removed?

**Review by avivace — CHANGES_REQUESTED** [review]
Looks great! I think the JSON just needs the trailing comma to be removed and it should validate

File: entries/grub-glide/game.json (hunk 1 above)
Comment (Exetric) [reply entries/grub-glide/game.json]: Just fixed the issue, thanks for helping point it out!

**Review by Exetric — COMMENTED** [review]
I removed the trailing comma so the JSON should work fine now.

**Review by avivace — APPROVED** [review]
Looks good! Thanks

**Pull request merged**